actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorEntry.java

```diff
@@ -296,13 +301,4 @@ public synchronized void clearTimers()
         }
         return Task.done();
     }
-
-    @Override
-    public String toString()
```

Comment:
in the end it was nicer to use the default toString implementation as it caused a stackoverflow for stateless actors.